kfp/website-ingestor/ingestion-pipeline-website-local.py

```diff
@@ -0,0 +1,446 @@
+import weaviate
+import os
+from langchain_community.llms import VLLMOpenAI
+from langchain_community.embeddings import HuggingFaceEmbeddings
+
+
+
+# Set up logging
+import logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+# Function to get Weaviate client
+def get_weaviate_client():
+    """Get the Weaviate client."""
+    logger.info("Fetching Weaviate client...")
+    weaviate_api_key = os.getenv("WEAVIATE_API_KEY")
+    weaviate_host = os.getenv("WEAVIATE_HOST")
+    weaviate_port = os.getenv("WEAVIATE_PORT")
+
+    if not all([weaviate_api_key, weaviate_host, weaviate_port]):
+        logger.error("Weaviate config not present. Check host, port, and API key.")
+        exit(1)
+
+    auth_config = weaviate.auth.AuthApiKey(api_key=weaviate_api_key)
+    return weaviate.Client(
+        url=f"{weaviate_host}:{weaviate_port}",
+        auth_client_secret=auth_config,
+    )
+
+
+def get_top_records(index_name, limit=10):
```

Comment:
Are we using this for anything or just testing?

Reply:
This is only for testing